docs/tutorials/basics/frames.rst

```diff
@@ -0,0 +1,70 @@
+Frames, ticks, titles, and labels
+-----------------------
+
+Setting frame, ticks, title, etc., of the plot is handled by the :gmt-module:`-B <basemap#b>` parameter that most plotting modules contain like :gmt-module:`basemap`.
+
+Plot Frame
+~~~~~~~~~~
+
+By default, GMT does not add a frame to your plot. For example, we can plot
+the coastlines of the world with a Mercator projection:
+
+
+.. gmtplot::
+
+   gmt begin frames png
```

Comment:
Good opportunity to introduce one-liners?

Reply:
I don't think so. This is a tutorial for beginners. I would prefer to keep it as simple as possible. I'll even add the title on a separate line. This way, it's easier for them to copy and paste into their own script."